units/systemd-binfmt.service.in

```diff
@@ -19,6 +19,7 @@ After=proc-sys-fs-binfmt_misc.mount
 After=local-fs.target
 Before=sysinit.target shutdown.target
 ConditionPathIsReadWrite=/proc/sys/
+ConditionVirtualization=!lxc
```

Comment:
Why not to simply use `ConditionPathIsReadWrite=/proc/sys/fs/binfmt_misc` ?
See #25693 .